Sources/App/Views/Search/SearchShow+Model.swift

```diff
@@ -68,3 +68,16 @@ enum SearchShow {
     }
 
 }
+
+extension String {
+    func filteredForDisplay() -> String {
```

Comment:
Nit: I'd call this `sanitized()` instead, matching out `Search.sanitize()` function.